app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -200,14 +217,27 @@ def floating_ips
 
   def volumes
     collector.volumes.each do |vol|
+      az_name = vol&.dig(:zone, :name)
+      byebug
+      attachments = vol&.dig(:volume_attachments)
+      bootable = attachments.any?
+      if bootable
+        attachments.each do |vol_attach|
+          bootable = vol_attach[:type] == "boot"
+          break if bootable
+        end
+      end
+      byebug
       persister.cloud_volumes.build(
-        :ems_ref       => vol[:id],
-        :name          => vol[:name],
-        :status        => vol[:status],
-        :creation_time => vol[:created_at],
-        :description   => 'IBM Cloud Block-Storage Volume',
-        :volume_type   => vol[:type],
-        :size          => vol[:capacity]&.gigabytes
+        :ems_ref              => vol[:id],
+        :name                 => vol[:name],
+        :status               => vol[:status],
+        :creation_time        => vol[:created_at],
+        :description          => 'IBM Cloud Block-Storage Volume',
+        :volume_type          => vol[:type],
+        :size                 => vol[:capacity]&.gigabytes,
+        :bootable             => bootable,
+        :availability_zone_id => persister.availability_zones.lazy_find(az_name)
```

Comment:
This should be `:availability_zone => persister.availability_zones.lazy_find(az_name)`

Reply:
fixed